Merge styles in a single file

kucho opened the pull request.

DeveLawyer commented:
El mismo fix del icon se requiere en el sandwich en vista mobile

DeveLawyer commented:
Veo un posible responsive issue en signup2, en mobile sale scroll horizontal:
![Screen Shot 2020-04-08 at 14 24 38](https://user-images.githubusercontent.com/52726455/78824959-d4cca400-79a4-11ea-989f-25945efdbfa2.png)

DeveLawyer commented:
Al clickar "Software engineer" te debe mandar al job-detail view:
![Screen Shot 2020-04-08 at 14 29 36](https://user-images.githubusercontent.com/52726455/78825434-98e60e80-79a5-11ea-90c9-1bd8eb990a23.png)

DeveLawyer commented:
Creo que falta añadir este par de íconos en el card de job-details:
![Screen Shot 2020-04-08 at 14 41 59](https://user-images.githubusercontent.com/52726455/78826536-460d5680-79a7-11ea-9cb2-d38ea93f263c.png)

DeveLawyer commented:
Falta completar el href de este link:
![Screen Shot 2020-04-08 at 14 45 27](https://user-images.githubusercontent.com/52726455/78826811-c5028f00-79a7-11ea-96ca-5d8545c59b01.png)

kucho commented:
> El mismo fix del icon se requiere en el sandwich en vista mobile

Fixed!



> Veo un posible responsive issue en signup2, en mobile sale scroll horizontal:
> ![Screen Shot 2020-04-08 at 14 24 38](https://user-images.githubusercontent.com/52726455/78824959-d4cca400-79a4-11ea-989f-25945efdbfa2.png)

Good eye! Fixed!



> Al clickar "Software engineer" te debe mandar al job-detail view:
> ![Screen Shot 2020-04-08 at 14 29 36](https://user-images.githubusercontent.com/52726455/78825434-98e60e80-79a5-11ea-90c9-1bd8eb990a23.png)

Done, thank you!



> Creo que falta añadir este par de íconos en el card de job-detail:
> ![Screen Shot 2020-04-08 at 14 41 59](https://user-images.githubusercontent.com/52726455/78826536-460d5680-79a7-11ea-9cb2-d38ea93f263c.png)

Nice catch! Done!

> Falta completar el href de este link en job-detail:
> ![Screen Shot 2020-04-08 at 14 45 27](https://user-images.githubusercontent.com/52726455/78826811-c5028f00-79a7-11ea-96ca-5d8545c59b01.png)

Solved, sir!
